optaplanner-examples/src/main/java/org/optaplanner/examples/conferencescheduling/persistence/ConnectionFollowRedirects.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// credits for https://www.cs.mun.ca/java-api-1.5/guide/deployment/deployment-guide/upgrade-guide/article-17.html
+
+package org.optaplanner.examples.conferencescheduling.persistence;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.net.URLConnection;
+
+public class ConnectionFollowRedirects {
+
+    private URLConnection connection;
+    private boolean isRedirect;
+    private int redirects = 0;
+
+    public ConnectionFollowRedirects(String url) throws IOException {
```

Comment:
Do we need a redirect on every URL or only on the first of the rest api?

Reply:
We need it on every URL, since the url references in some of the data returns 301 status code and others don't.

For example: the first data entry in this endpoint https://dvbe18.confinabox.com/api/conferences/DVBE18/speakers contains a reference to the speaker data http://dvbe18.confinabox.com/api/conferences/DVBE18/speakers/f8caf20b7f6f226bf0637bc8e077d95161561439 which returns a response with code 301 (it redirects to https).